ow/launch/common.launch

```diff
@@ -77,19 +81,13 @@
   <!-- ====== simulate changes to the terrain caused by the grinder ======= -->
   <node name="modify_terrain_grinder" pkg="ow_dynamic_terrain" type="modify_terrain_grinder_pub.py" output="screen"/>
 
-  <!-- == start rviz ==================== -->
-  <!--group unless="$(arg arm_sim_enable)">
-    <node name="rviz" pkg="rviz" type="rviz"
-      args="-d $(find ow_lander)/config/lander.rviz" output="screen"/>
-  </group-->
-
   <!-- == Start rqt with a short delay so most topics will be visible and we won't need to refresh widgets ====== -->
   <!-- Starting without a .perspective file will reload the last used configuration. -->
   <!--node pkg="rqt_gui" type="rqt_gui" name="rqt_gui"
     launch-prefix="bash -c 'sleep 5; $0 $@' "/-->
   <node pkg="rqt_gui" type="rqt_gui" name="rqt_gui"
     launch-prefix="bash -c 'sleep 5; $0 $@' "
-    args="&#45;&#45;perspective-file $(find ow)/config/default.perspective">
+    args="&#45;&#45;perspective-file $(find ow)/config/default.perspective" if="$(arg rqt_gui)">
```

Comment:
#nit I like having the conditional on the first line after "<node" since everything that follows is dependent upon it.

Reply:
addressed